packages/js/tracking-jsdoc/README.md

```diff
@@ -0,0 +1,83 @@
+# woo-tracking-jsdoc
```

Comment:
❓ 

In `.jsdocrc.json` I see `jsdoc-plugin-intersection` as well as `"recurse": true`

Not sure if we can remove them. If they are useful we could add it to the Readme as well?

Reply:
We cannot remove them, as they are needed. `jsdoc-plugin-intersection` - is a plugin that lets `jsdoc` parse types intersections. and `recurse` is a CLI argument that makes `jsdoc` parse the code recursively.

I wouldn't mention them in `woo-tracking-jsdoc` as those configs have nothing to do with tracking, and are not related at all to this plugin.
Potentially, I could add/mention them in a potential new package mentioned in https://github.com/woocommerce/grow/pull/6#issuecomment-1137227272

Reply:
Problem of not mentioning anything is that when a developer implements the tool he wont add those into `.jsdocrc.json` creating errors. 

As far as I see, in the README.md should be instructions about how to have the tool running.

Reply:
But the tool described here is `woo-tracking-jsdoc` plugin, not the entire `jsdoc`.

In Google Listing & Ads README.md, we do not mention how to add AutomateWoo to WooCommerce, or how to add variable product to WC products list.

Reply:
This PR does not deliver a "tool to generate all the docs for an extension"  (yet, we could implement that in the following PRs). This PR only moves one `jsdoc` template and a few pluginsfor tracking docs onyl.

Reply:
> we could implement that in the following PRs

I can buy that :)

Reply:
> Yes, I got your point about that. Since in theory are conceptually separated things.
> 
> But in the reality what I experienced was that after installing `woocommerce-grow-tracking-jsdoc` I could not track anything since I needed dependencies and configs that doesn't appear in the Readme (see intersection, recurse and jsdoc-advanced-types-plugin and jsdoc itself).
> 
> By adding those to the repo I just run `doc:tracking` and go the only thing that (maybe) I need to do is to add the provided config in the readme.

In reality, they have also separated things.
If you would have a repo whose source code is already parseable by `jsdoc`, adding `woocommerce-grow-tracking-jsdoc`. If you create a repo with a single js file with a simple JSDoc in it, it will work.

BTW `jsdoc` is added as a peer dependency, so you get a warning if you don't have it. (The amount of warnings we get in GLA, and warning blindness it introduces, are a separate topics, beyond the scope of PR ;) )

Reply:
What I think we could add to this readme is
> 0. Set up jsdoc, so it parses your source code. 

From the perspective of this plugin that would be enough to make `docs:tracking` work once you just added it.

As well, we cannot anticipate all the JS and JSDoc parser problems one may get with their source code.

Reply:
So my main comment is:

Are we (grow) using JSDOC for anything besides tracking the docs? --> I think no
Are we (grow) planning to use JSDOC for anything besides tracking the docs? --> I think no, neither

But I'm fine if we have 2 different repos. One for tracking only and other for loading jsdoc base deps